packages/app/src/app/pages/NewDashboard/utils/get-child-collections.js

```diff
@@ -0,0 +1,15 @@
+import getDirectChildren from './get-direct-children';
+
+export default function getChildCollections(folders, path = '/') {
+  const foldersByPath = {};
+
+  folders.forEach(collection => {
+    foldersByPath[collection.path] = collection;
+  });
```

Comment:
This can be simplified by using `reduce`

```suggestion
  const foldersByPath = folders.reduce(
    (pathedFolders, folder) => ({
      ...pathedFolders,
      [folder.path]: folder,
    }),
    {}
  );
```

Reply:
Oh, nice!